src/sentry/dynamic_sampling/latest_release_booster.py

```diff
@@ -161,3 +164,47 @@ def add_boosted_release(project_id: int, release_id: int, environment: Optional[
         datetime.utcnow().replace(tzinfo=UTC).timestamp(),
     )
     redis_client.pexpire(cache_key, BOOSTED_RELEASE_TIMEOUT * 1000)
+
+
+@dataclass(frozen=True)
+class BoostedRelease:
+    version: str
+    environment: Optional[str]
+    platform: Platform
+    timestamp: float
+
+
+def _get_project_platform_from_release(release: Release, project_id: int) -> Optional[Project]:
+    release_projects = release.projects.filter(id__in=[project_id])
+    return release_projects[0] if len(release_projects) > 0 else None
+
+
+def get_boosted_releases_augmented(project_id: int, limit: int) -> List[BoostedRelease]:
+    """
+    Returns a list of boosted releases augmented with additional information such as release version and platform.
+    """
+    cached_boosted_releases = get_and_expire_boosted_releases(project_id)
+    if not cached_boosted_releases:
+        return []
+
+    # We get the ids of the last limit-th releases.
+    boosted_releases_ids = [release_id for (release_id, _, _) in cached_boosted_releases[-limit:]]
```

Comment:
can you show me data structure for `cached_boosted_releases` i can probobly help to simplify it.

Reply:
https://github.com/getsentry/sentry/blob/e2a549c84d6f841fb5865d5f85e50d4cb9b41baf/src/sentry/dynamic_sampling/latest_release_booster.py#L139-L139